From b3d2e08ec9d626956281ec811a864f3a1fe09d20 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Fri, 21 Dec 2012 17:27:30 +0530
Subject: [PATCH 8/8] NFS: Fix build breakge when CONFIG_NFS_FSCACHE is not
 defined

Commit de242c0b (NFS: Use FS-Cache invalidation) introduced
a build breakge for the case where CONFIG_NFS_FSCACHE is not
defined.

...
fs/nfs/inode.c: In function 'nfs_invalidate_mapping':
fs/nfs/inode.c: error: implicit declaration of function 'nfs_fscache_wait_on_invalidate'
make[2]: *** [fs/nfs/inode.o] Error 1

Add an empty function similar to fix this.

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 fs/nfs/fscache.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/fs/nfs/fscache.h b/fs/nfs/fscache.h
index 277b027..e06d4f4 100644
--- a/fs/nfs/fscache.h
+++ b/fs/nfs/fscache.h
@@ -223,6 +223,8 @@ static inline void nfs_readpage_to_fscache(struct inode *inode,
 
 static inline void nfs_fscache_invalidate(struct inode *inode) {}
 
+static inline void nfs_fscache_wait_on_invalidate(struct inode *inode) {}
+
 static inline const char *nfs_server_fscache_state(struct nfs_server *server)
 {
 	return "no ";
-- 
1.7.10.4

